### ISSUE TITLE
NFR: Establish Road Choice Websites for Mexico and Canada 

### ISSUE THREAD

**Issue by ZoeBeykirch** [hlxsites/vg-roadchoice-com#222]
**Dependency:** https://github.com/hlxsites/vg-roadchoice-com/issues/223 

### Description
As the Road Choice business owner, I want to create new multi-language dedicated websites for our Canadian and Mexican markets, leveraging the existing structure and content from the [USA website](https://roadchoice.com/). The goal is to efficiently expand our online presence to Canada and Mexico while maintaining a cohesive brand identity across all websites. 

### Acceptance Criteria
- Sharepoint folders are to be created for each of Mexico and Canada
- New market sites shall be configured to allow all of the same component blocks from the main website to be used. 
- The [USA Road Choice website](https://roadchoice.com/) shall be the parent site, mirroring the website page structure for the Mexico and Canadian market sites.
- The management of some product data catalogs will require separate excel sheets for each market since the offerings may differ.
- The Road Choice Canadian website will require a French and English version. 
- The Road Choice Mexico site will first only require Spanish, then have the possibility to add English in the future if needed. 
- Hardcoded items (ex. Dealer Locator) are to be translated manually and picked up in other tickets  

- [x] Sharepoint folders have been created for both Mexico and Canada.
- [x] Content mapping and translations completed by CMS team
- [ ] Hardcoded items to be taken up by dev team 

### Next Steps:
This is the preview url of the new Road Choice repo: https://main--vg-roadchoice-com--volvogroup.aem.page/

- [ ] From here we need the QA team ([@girish.digrajkar](https://ncchat.slack.com/team/U017YJ79KUN)) to test the website and find issues/inconsistencies. For this our current preview can easily be used to compare:
https://main--vg-roadchoice-com--hlxsites.hlx.page/
- [ ] Meanwhile the Content team ([@Kannan C](https://ncchat.slack.com/team/U066LB3DBJM)) needs:
1. First check why we have this online: https://www.roadchoice.ca/fr-ca/
(From Alexis understanding you are handling the DNS configs of the domains.)
2. Next you need to validate the Road Choice content for Mexico and Canada.

- [ ] After the above steps are done the QA team can start testing the new Road Choice Mexico website on preview:
https://main--roadchoice-mx--volvogroup.aem.page/
And the new Road Choice Canada website on preview:
https://main--roadchoice-ca--volvogroup.aem.page/
At the moment there is a bug breaking most of the functionality of the website which I plan to have a fix by tomorrow. After this all the testing can start.

### Out of Scope
- Any language translations for market sites. This will be handled separately by CMS, and market sites can be delivered in English

**Comment by mollyaljenkins** [comment]
**Questions** 
- Do we start already with adding content to different language folders (spanish) for the Mexican website? 

**Need to establish:**
- OneTrust 
- Separate ID for each website 
- Any third party tools?


**User story:** 
As a user of RoadChoice's website, I want to be able to easily change the language of the website content using a feature in the header, so that I can engage with the content in my chosen language.

**Comment by mollyaljenkins** [comment]
This ticket is on hold until the results of the spike from #223

**Comment by ZoeBeykirch** [comment]
@janedbarra What product data will you expect to be different from US sites on the new market sites?

**Comment by janedbarra** [comment]
The .CA site will very closely mirror the current .com site. The only products that offered in the US only are DEF and Mud Flaps, so these would not be included in the .ca site. There is one program (Coolant) where part numbers differ b/t US and CA.  All other product offerings will be the same across .ca and .com. 

For dealer locator, I would like this to be present on all 3 sites, but only show the respective country. 

For the rest of the content pages, they would be duplicated exactly b/t .CA and .com (other than omission of DEF and Mud Flap resources and mentions on the respective category pages). 

For the .MX site For Mexico, the following category pages and part data should be copied over:
•	Engine Cooling - (https://www.roadchoice.com/part-category/engine-cooling) and part listings for Radiators, Charge Air Coolers, Fan Clutches, Fan Clutch Kits, and Fan Clutch Valves

•	Suspension - (https://www.roadchoice.com/part-category/suspension)and part listings for Air Springs

•	Engine - (https://www.roadchoice.com/part-category/engine) and part listings for V-Belts

•	Electrical - (https://www.roadchoice.com/part-category/electrical) and part listings for Batteries

We will also need the ability to add to this as the market opportunity expands.

**Comment by BeckyMedlin** [comment]
@mollyaljenkins  Please review @janedbarra's comments above and advise if anything further is needed. I believe this can be unblocked.

**Comment by janedbarra** [comment]
I would want it to be focused on the country that's visited on load. For example, a user on roadchoice.ca would show Canada predominately, but they could search for US retailers as well.

**Comment by mollyaljenkins** [comment]
next steps are to investigate content folders via the spike https://github.com/hlxsites/vg-roadchoice-com/issues/241

**Comment by Joesturbo** [comment]
Check Darin @adobe on status of RC sites and upgrade to Helix5

**Comment by mollyaljenkins** [comment]
When this is complete we can start on items such as #233

**Comment by Kannan-CM** [comment]
Hi @mollyaljenkins ,

The Translations are complete for fr-ca and es-mx

**Comment by BeckyMedlin** [comment]
@mollyaljenkins and @Kannan-CM Can this ticket be closed as completed?

**Comment by Kannan-CM** [comment]
Hi @BeckyMedlin, 
We have done only the content translation. So, I'll leave it to @mollyaljenkins for final call.

**Comment by BeckyMedlin** [comment]
Per @Kannan-CM on 6/24/2024: 
The ticket had 3 points:
1. Sharepoint folders have been created for both Mexico and Canada. - COMPLETE
2. Kannan and CMS team to take up content mapping and translations. - COMPLETE
3. Hardcoded items will be taken up separately.

Point 1 and 2 are done.
Point number 3 needs to be completed by Netcentric.

We have copied all the folders and Content from RC English Site and translated the Product Category Pages, Blogs, and Pages under Resources.

**Comment by BeckyMedlin** [comment]
Need clarification on what the hard coded items are - Excel files, code (text). We also need input from @dkuntze on status of Road Choice Helix5 upgrade. Dependent on #223 @ZoeBeykirch

**Comment by BeckyMedlin** [comment]
Domain transfer for roadchoice.mx requested on 7/9/24.

**Comment by ZoeBeykirch** [comment]
After all repos are under the same branch, then we can unblock this.

**Comment by girishdigrajkar** [comment]
https://main--vg-roadchoice-com--volvogroup.aem.page/ has been tested by all QAs with no outstanding issues.
